modules/isolation_segment/firewalls.tf

```diff
@@ -0,0 +1,129 @@
+resource "google_compute_firewall" "isoseg-cf-internal" {
+  count = "${var.count * var.with_firewalls}"
+
+  name     = "${var.env_name}-isoseg-cf-internal"
+  network  = "${var.network}"
+  priority = 997
+
+  allow {
+    protocol = "icmp"
+  }
+
+  allow {
+    protocol = "tcp"
+  }
+
+  allow {
+    protocol = "udp"
+  }
+
+  source_tags = ["isoseg-${var.env_name}"]
+  target_tags = ["isoseg-${var.env_name}"]
+}
+
+resource "google_compute_firewall" "isoseg-cf-ingress" {
+  count = "${var.count * var.with_firewalls}"
+
+  name    = "${var.env_name}-isoseg-cf-ingress"
+  network = "${var.network}"
+
+  source_ranges = ["${var.pas_subnet_cidr}"]
+  target_tags   = ["isoseg-${var.env_name}"]
+  priority      = 998
+
+  allow {
+    protocol = "tcp"
+    ports    = ["1801", "8853"]
+  }
+}
+
+resource "google_compute_firewall" "isoseg-block-cf-ingress" {
+  count = "${var.count * var.with_firewalls}"
+
+  name    = "${var.env_name}-isoseg-block-cf-ingress"
+  network = "${var.network}"
+
+  direction   = "INGRESS"
+  source_tags = ["cf-${var.env_name}"]
+  target_tags = ["isoseg-${var.env_name}"]
+  priority    = 999
+
+  deny {
+    protocol = "icmp"
+  }
+
+  deny {
+    protocol = "tcp"
+  }
+
+  deny {
+    protocol = "udp"
+  }
+}
+
+resource "google_compute_firewall" "cf-isoseg-ingress" {
+  count = "${var.count * var.with_firewalls}"
+
+  name    = "${var.env_name}-cf-isoseg-ingress"
+  network = "${var.network}"
+
+  source_tags = ["isoseg-${var.env_name}"]
+  target_tags = ["cf-${var.env_name}"]
+  priority    = 998
+
+  allow {
+    protocol = "tcp"
+
+    ports = [
+      "3000",
+      "3457",
+      "4003",
+      "4103",
+      "4222",
+      "4443",
+      "8080",
+      "8082",
+      "8084",
```

Comment:
This port is used for the diego fileserver: https://github.com/pivotal-cf/p-runtime/blob/master/variables/srt.yml#L42

There are some ports that are opened for when you are deploying the Small Footprint PAS which is the tiny version of PAS and has different port requirements because of a different colocation strategy.

Reply:
Thank you @genevieve this is very helpful!